### PR TITLE
initial push of Darnah Flood data story

### PR DESCRIPTION
**Pull request by acblackford**
## Why are you creating this Pull Request?

- [Adding Datasets or Stories](?title=Content%3A%20%3Cname%3E&expand=1&template=content.md)
- [Version Release](?title=Deploy%20vX.X.X&expand=1&template=version_release.md)
- [Other](?expand=1&template=default.md)

### CODE REVIEW

**Comment by j08lue** [comment]
Just a quick reminder of this undocumented not-yet-feature, @acblackford: You can post your MDX file content into this MDX editor here to do live edits: https://deploy-preview-372--visex.netlify.app/sandbox/mdxeditor

You just need to push all images you want to use to your branch here, so the editor can pull them. And when you are done editing, you need to manually update your PR with the updated MDX from the editor. Please let me know if you would like to know more, then I can give you a brief walk-through.

**Comment by jerikac** [comment]
@j08lue or @hanbyul-here what needs to happen to push this to prod? is there a system we can set up for upcoming VEDA Data Stories to streamline the process?

File: stories/darnah-flood.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+</Block>
+
+
+<ScrollytellingBlock>
+  <Chapter
+    center={[17.29046905780905,38.5666921324378]}
+    zoom={6}
+    datasetId='darnah-flood'
+    layerId='darnah-gpm'
+    datetime='2023-09-01'
```
Comment (hanbyul-here) [inline stories/darnah-flood.stories.mdx:78]: This datetime does not match the datetime in the paragraph (the paragraph says September 4th) + As far as I can check from the dataset explorer https://deploy-preview-372--visex.netlify.app/data-catalog/darnah-flood/explore?projection=mercator%7C%7C&basemapid=satellite&datetime=2023-09-07T00%3A00%3A00.000Z The oldest data that can be retrieved from our endpoint is September 7th. What is the intended visual for this chapter (and below)?

**Comment by anayeaye** [comment]
Heads up, please do not merge this PR until the assets and STAC metadata have been published to production https://github.com/NASA-IMPACT/veda-data/issues/149

**Review by sandrahoang686 — APPROVED** [review]
It has been migrated over to openveda and should be unblocked

**Pull request merged**